airflow/migrations/versions/d38e04c12aa2_add_serialized_dag_table.py

```diff
@@ -45,6 +47,37 @@ def upgrade():
     op.create_index(   # pylint: disable=no-member
         'idx_fileloc_hash', 'serialized_dag', ['fileloc_hash'])
 
+    conn = op.get_bind()
+    if conn.dialect.name == "mysql":
+        conn.execute("SET time_zone = '+00:00'")
+        cur = conn.execute("SELECT @@explicit_defaults_for_timestamp")
+        res = cur.fetchall()
+        if res[0][0] == 0:
+            raise Exception(
+                "Global variable explicit_defaults_for_timestamp needs to be on (1) for mysql"
+            )
+
+        op.alter_column(
+            table_name="serialized_dag",
+            column_name="last_updated",
+            type_=mysql.TIMESTAMP(fsp=6),
+        )
+    else:
+        # sqlite and mssql datetime are fine as is.  Therefore, not converting
+        if conn.dialect.name in ("sqlite", "mssql"):
+            return
+
+        # we try to be database agnostic, but not every db (e.g. sqlserver)
+        # supports per session time zones
+        if conn.dialect.name == "postgresql":
+            conn.execute("set timezone=UTC")
+
+        op.alter_column(
+            table_name="serialized_dag",
+            column_name="last_updated",
+            type_=sa.TIMESTAMP(timezone=True),
+        )
+
```

Comment:
@coufon  - Should work now

![image](https://user-images.githubusercontent.com/8811558/62814112-9a092800-bb06-11e9-8c5a-73863c2b58ff.png)